superset/assets/cypress/integration/explore/visualizations/area.js

```diff
@@ -0,0 +1,82 @@
+describe('Area', () => {
+  const AREA_FORM_DATA = {
+    datasource: '2__table',
+    viz_type: 'area',
+    slice_id: 48,
+    url_params: {},
+    granularity_sqla: 'year',
+    time_grain_sqla: 'P1D',
+    time_range: '1960-01-01 : now',
+    metrics: ['sum__SP_POP_TOTL'],
+    adhoc_filters: [],
+    groupby: [],
+    limit: '25',
+    timeseries_limit_metric: null,
+    order_desc: true,
+    contribution: false,
+    row_limit: 50000,
+    show_brush: 'auto',
+    show_legend: true,
+    line_interpolation: 'linear',
+    stacked_style: 'stack',
+    color_scheme: 'bnbColors',
+    rich_tooltip: true,
+    show_controls: false,
+    x_axis_label: '',
+    bottom_margin: 'auto',
+    x_ticks_layout: 'auto',
+    x_axis_format: 'smart_date',
+    x_axis_showminmax: false,
+    y_axis_format: '.3s',
+    y_axis_bounds: [null, null],
+    y_log_scale: false,
+    rolling_type: 'None',
+    comparison_type: 'values',
+    resample_how: null,
+    resample_rule: null,
+    resample_fillmethod: null,
+    annotation_layers: [],
```

Comment:
I've been trying to reduce the number for form_data params that are hardcoded, just because we'll need to change them when they change. What do you think about reducing this list to the form_data fields that are necessary for the chart to load (removing those that are null).